tinkerbell/hegel/templates/deployment.yaml

```diff
@@ -44,6 +44,9 @@ spec:
           ports:
             - containerPort: {{ .Values.deployment.port }}
               name: {{ .Values.deployment.portName }}
+              {{- if .Values.deployment.hostPortEnabled }}
+              hostPort: {{ .Values.deployment.port }}
+              {{- end }}
```

Comment:
Curious, is there a known use case for needing this? Kubernetes has some general guidance to not use `hostPort`. [ref](https://kubernetes.io/docs/concepts/configuration/overview/#:~:text=Don%27t%20specify%20a,default%20protocol.)